drivers/bus/acpi/CMakeLists.txt

```diff
@@ -1,214 +1,2 @@
-    acpica/resources/rsaddr.c
-    acpica/resources/rscalc.c
-    acpica/resources/rscreate.c
-    # acpica/resources/rsdump.c
-    # acpica/resources/rsdumpinfo.c
-    acpica/resources/rsinfo.c
-    acpica/resources/rsio.c
-    acpica/resources/rsirq.c
-    acpica/resources/rslist.c
-    acpica/resources/rsmemory.c
-    acpica/resources/rsmisc.c
-    acpica/resources/rsserial.c
-    acpica/resources/rsutils.c
-    acpica/resources/rsxface.c
-    acpica/tables/tbdata.c
-    acpica/tables/tbfadt.c
-    acpica/tables/tbfind.c
-    acpica/tables/tbinstal.c
-    acpica/tables/tbprint.c
-    acpica/tables/tbutils.c
-    acpica/tables/tbxface.c
-    acpica/tables/tbxfload.c
-    acpica/tables/tbxfroot.c
-    acpica/utilities/utaddress.c
-    acpica/utilities/utalloc.c
-    acpica/utilities/utascii.c
-    acpica/utilities/utbuffer.c
-    acpica/utilities/utcache.c
-    acpica/utilities/utcksum.c
-    # acpica/utilities/utclib.c
-    acpica/utilities/utcopy.c
-    # acpica/utilities/utdebug.c
-    acpica/utilities/utdecode.c
-    acpica/utilities/utdelete.c
-    acpica/utilities/uterror.c
-    acpica/utilities/uteval.c
-    # acpica/utilities/utfileio.c
-    acpica/utilities/uthex.c
-    acpica/utilities/utids.c
-    acpica/utilities/utinit.c
-    acpica/utilities/utlock.c
-    acpica/utilities/utmath.c
-    acpica/utilities/utmisc.c
-    acpica/utilities/utmutex.c
-    acpica/utilities/utnonansi.c
-    acpica/utilities/utobject.c
-    acpica/utilities/utosi.c
-    acpica/utilities/utownerid.c
-    acpica/utilities/utpredef.c
-    # acpica/utilities/utprint.c
-    # acpica/utilities/utresdecode.c
-    acpica/utilities/utresrc.c
-    acpica/utilities/utstate.c
-    acpica/utilities/utstring.c
-    acpica/utilities/utstrtoul64.c
-    acpica/utilities/utstrsuppt.c
-    # acpica/utilities/utuuid.c
-    acpica/utilities/uttrack.c
-    acpica/utilities/utxface.c
-    acpica/utilities/utxferror.c
-    acpica/utilities/utxfinit.c
-    acpica/utilities/utxfmutex.c)
-
-list(APPEND ACPICA_PCH_SKIP_SOURCE
-    acpica/utilities/utexcep.c
-    acpica/utilities/utglobal.c)
-
-add_library(acpica
-    ${ACPICA_SOURCE}
-    ${ACPICA_PCH_SKIP_SOURCE})
-
-add_pch(acpica acpica/include/acpi.h "${ACPICA_PCH_SKIP_SOURCE}")
-set_target_properties(acpica PROPERTIES COMPILE_DEFINITIONS "ACPI_USE_LOCAL_CACHE;ACPI_USE_SYSTEM_INTTYPES")
-add_dependencies(acpica bugcodes xdk)
-
-list(APPEND ACPI_SOURCE
-    busmgr/bus.c
-    busmgr/button.c
-    busmgr/power.c
-    busmgr/utils.c
-    busmgr/system.c
-    osl.c
-    acpienum.c
-    eval.c
-    interface.c
-    pnp.c
-    power.c
-    buspdo.c
-    main.c)
-
-add_library(acpi MODULE
-    ${ACPI_SOURCE}
-    acpi.rc)
-
-set_module_type(acpi kernelmodedriver)
-target_link_libraries(acpi wdmguid acpica memcmp strtol ${PSEH_LIB})
-add_importlibs(acpi ntoskrnl hal)
-add_pch(acpi precomp.h ACPI_SOURCE)
-add_cd_file(TARGET acpi DESTINATION reactos/system32/drivers NO_CAB FOR all)
-add_driver_inf(acpi acpi.inf)
-
+add_subdirectory(acpi)
```

Comment:
Uhm... not really a fan of `.../acpi/acpi/` directory structure. I would still vote for keeping the acpi bus directory for just the bus driver (old and new), and moving cmbatt to batteries.

Reply:
There's going to be other drivers under acpi/ 
maybe acpi/ can be under root of drivers but still there's going to be a handful of more drivers smacked in